src/sage/matroids/oriented_matroids/oriented_matroid.py

```diff
@@ -0,0 +1,737 @@
+        ]
+
+        return Poset((els, rels), cover_relations=False, facade=facade)
+
+    def is_simplicial(self):
+        r"""
+        Return if the oriented matroid is simplicial.
+
+        An oriented matroid is *simplicial* if every tope is simplicial.
+
+        .. SEEALSO::
+
+            :meth:`~sage.matroids.oriented_matroids.signed_subset_element.SignedSubsetElement.is_simplicial`
+        """
+        for t in self.topes():
+            if not t.is_simplicial():
+                return False
+        return True
+
+    def is_acyclic(self):
+        r"""
+        Return if oriented matroid is acyclic.
+
+        A covector oriented matroid is *acyclic* if there exists a positive
+        tope where a *positive tope* is defined as a tope with no
+        negative part.
+        """
+        for t in self.topes():
+            if len(t.negatives()) == 0:
+                return True
+        return False
+
+    def deletion(self, change_set):
+        r"""
+        Return a covector oriented matroid of a deletion.
+
+        Let `M = (E, \mathcal{L})` be an oriented matroid over a set `E`
+        and a set of covectors `\mathcal{L}`. Given `A \subseteq E`, the
+        *deletion* is the (covector) oriented matroid
+        `M\backslash A = (E \backslash A, \mathcal{L} \backslash A)` where
+
+        .. MATH::
+
+            \mathcal{C} \backslash A = \left\{ X\mid_{E \backslash A} : X \in \mathcal{C}\right\}
+
+        EXAMPLES::
+
+            sage: from sage.matroids.oriented_matroids.oriented_matroid import OrientedMatroid
+        """
+        if change_set in self.groundset():
+            change_set = set([change_set])
+        else:
+            change_set = set(change_set)
+
+        from sage.matroids.oriented_matroids.oriented_matroid import deep_tupler
+        groundset = set(self.groundset()).difference(change_set)
+        groundset = deep_tupler(groundset)
+        data = []
+        for c in self.covectors():
+            p = tuple(c.positives().difference(change_set))
+            n = tuple(c.negatives().difference(change_set))
+            z = tuple(c.zeros().difference(change_set))
+            data.append((p, n, z))
+        data = deep_tupler(data)
+
+        from sage.matroids.oriented_matroids.oriented_matroid import OrientedMatroid
+        return OrientedMatroid(data, key='covector', groundset=groundset)
+
+    def restriction(self, change_set):
+        r"""
+        Return a covector oriented matroid of a restriction.
+
+        Given an oriented matroid `M = (E, \mathcal{L})` where `E` is a
+        set and `\mathcal{L}` is the set of covectors. Given
+        `A \subseteq E`, the *restriction* is the (covector) oriented
+        matroid `M / A = (E \backslash A, \mathcal{C} / A)` where
+
+        .. MATH::
+
+            \mathcal{C} / A = \left\{ X\mid_{E \backslash A} : X \in \mathcal{C} \text{ and} A \subseteq X^0 \right\}
```

Comment:
```suggestion
        matroid `M / A = (E \backslash A, \mathcal{C} / A)`, where

        .. MATH::

            \mathcal{C} / A = \left\{ X\mid_{E \backslash A} : X \in \mathcal{C} \text{ and} A \subseteq X^0 \right\}.
```
What about `\setminus` instead of `\backslash`?